services/frontend-service/src/ui/components/button/ExpandButton.tsx

```diff
@@ -97,3 +97,93 @@ export const ExpandButton = (props: ExpandButtonProps): JSX.Element => {
         </div>
     );
 };
+
+export type EnvGroupExpandButtonProps = {
+    onClickSubmit: (shouldLockToo: boolean) => void;
+    onClickLock: () => void;
+    defaultButtonLabel: string;
+    disabled: boolean;
+};
+
+export const EnvGroupExpandButton = (props: EnvGroupExpandButtonProps): JSX.Element => {
```

Comment:
This one needs some tests.

Reply:
Merged with ExpandButton. Added one test case to test clicking it